WhaleSpotting/react-app/src/components/SightingMapInfo.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect, useState } from "react";
+import WeatherApiModel from "../apiModels/WeatherApiModel";
+import { Chosen } from "./Map";
+import "../styles/SightingMapInfo.scss";
+
+interface infoProps {
+    chosen: Chosen | undefined;
+}
+
+export default function SightingMapInfo({ chosen }: infoProps): JSX.Element {
+
+    const key = process.env.REACT_APP_WEATHER_API_KEY;
+    
+    const response = {
+        lon: chosen?.lon,
+        lat: chosen?.lat,
+        species: ["Whale", "Orca", "Dolphin"]
+    };
+
+    const [weatherData, setWeatherData] = useState<WeatherApiModel>();
+    async function fetchWeather(): Promise<WeatherApiModel | void> {
+        //eslint-disable-next-line
```

Comment:
What's this disable for?

Reply:
It's chosen! as we are sure that it can't be undefined in here and the linting shows a warning on that

Reply:
Ahh ok cool thanks :)